tasks/main.yml

```diff
@@ -24,6 +24,9 @@
     nginx_version: "{{ nginx_v | regex_replace('.*nginx version: nginx/([\\d.]+).*', '\\1') }}"
     nginx_openssl_version: "{{ nginx_v | regex_replace('.*built with OpenSSL ([\\d.]+[a-z]*).*', '\\1') }}"
 
+- include: selinux.yml
```

Comment:
I need to fix the bare `include`s above but could you make this `include_tasks`?

Reply:
Sure, I can do all of them